record/tombstones.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tsdb
+package record
```

Comment:
Does it make sense for the whole tombstone code to move? Or just the `Stones` type?

Reply:
Up for debate. For the WAL tailing code to work I think only `Stone` would need to move. I don't feel strongly about moving all of `tombstones.go` but in general I think the amount of code in the root package is problematic.